src/mirage_net.ml

```diff
@@ -53,8 +53,8 @@ module type S = sig
   val pp_error: error Fmt.t
   type t
   val disconnect : t -> unit Lwt.t
-  val write: t -> size:int -> (Cstruct.t -> int) -> (unit, error) result Lwt.t
-  val listen: t -> header_size:int -> (Cstruct.t -> unit Lwt.t) -> (unit, error) result Lwt.t
+  val write: t -> size:int -> (Bytes.t -> int) -> (unit, error) result Lwt.t
```

Comment:
```suggestion
  val write: t -> size:int -> (bytes -> int) -> (unit, error) result Lwt.t
```

I find the `bytes` more readable than `Bytes.t` -- but that's certainly a question of taste.